prototype_source/pt2e_quant_xpu_inductor.rst

```diff
@@ -0,0 +1,230 @@
+PyTorch 2 Export Quantization with Intel GPU Backend through Inductor
+==================================================================
+
+**Author**: `Yan Zhiwei <https://github.com/ZhiweiYan-96>`_, `Wang Eikan <https://github.com/EikanWang>`_, `Zhang Liangang <https://github.com/liangan1>`_, `Liu River <https://github.com/riverliuintel>`_, `Cui Yifeng <https://github.com/CuiYifeng>`_
+
+Prerequisites
+---------------
+
+-  `PyTorch 2 Export Post Training Quantization <https://pytorch.org/tutorials/prototype/pt2e_quant_ptq.html>`_
+-  `TorchInductor and torch.compile concepts in PyTorch <https://pytorch.org/tutorials/intermediate/torch_compile_tutorial.html>`_
+
+Introduction
+--------------
+
+This tutorial introduces XPUInductorQuantizer, which aims to serve quantized models for inference on Intel GPUs.
+It utilizes the PyTorch 2 Export Quantization flow and lowers the quantized model into the inductor.
+
+The Pytorch 2 Export Quantization flow uses `torch.export` to capture the model into a graph and perform quantization transformations on top of the ATen graph.
```

Comment:
Do we need to call it "PyTorch 2 Export Quantization flow" or can it be just "Export Quantization flow"?
```suggestion
The PyTorch 2 Export Quantization flow uses ``torch.export`` to capture the model into a graph and perform quantization transformations on top of the ATen graph.
```

Reply:
hi, @svekars , `PyTorch 2 Export` here should be a full description of `pt2e` in APIs like `prepare_pt2e, convert_pt2e`. Could we keep this just like x86InductorQuantizer here https://pytorch.org/tutorials/prototype/pt2e_quant_x86_inductor.html?

Reply:
Sounds good!